flink/flink-scala-parent/pom.xml

```diff
@@ -41,10 +41,19 @@
     <hive.version>2.3.4</hive.version>
     <hiverunner.version>4.0.0</hiverunner.version>
     <grpc.version>1.15.0</grpc.version>
+    <!-- Start from 1.15 flink is scala free (no scala version in its artifact) -->
+    <flink.library.scala.suffix>_${flink.scala.binary.version}</flink.library.scala.suffix>
 
     <flink.bin.download.url>https://archive.apache.org/dist/flink/flink-${flink.version}/flink-${flink.version}-bin-scala_${flink.scala.binary.version}.tgz</flink.bin.download.url>
   </properties>
 
+  <repositories>
+    <repository>
+      <id>flink-115-repo</id>
+      <name>flink-115-repo</name>
+      <url>https://repository.apache.org/content/repositories/orgapacheflink-1488/</url>
+    </repository>
+  </repositories>
```

Comment:
Flink 1.15 has been released, so all of the staging repositories for the release candidates have been taken down per ASF policy.

Note the 1.15.0 jar for `flink-streaming-java` on maven central: https://mvnrepository.com/artifact/org.apache.flink/flink-streaming-java/1.15.0 🙂

Reply:
Thanks for review @kbendick Let me fix it.

Reply:
No problem. Hope I'm not intruding. I work in the Apache Iceberg community mostly, and have been using some beta builds of this PR for Flink 1.15 testing.

Reply:
That's awesome, I am creating an iceberg tutorial in Zeppelin notebook, hope that will be helpful for iceberg community